packages/frontend/src/components/chart/configure/render/renderHover.ts

```diff
@@ -154,6 +155,16 @@ export function renderHover(
         rows.push(renderCBVRow(point.usdParts.cbv))
         rows.push(renderEBVRow(point.usdParts.ebv))
         rows.push(renderNMVRow(point.usdParts.nmv))
+      } else if (
+        state.view.chart.type === 'TokenDetailedTvlChart' &&
+        'balance' in point
```

Comment:
why do we need this "balance in point"?

Reply:
Since it's a union, it doesn't know which fields are present in this objects, and if we check that `balance` is present now we know for sure that this object is `TokenDetailedTvlChart`.